powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/GetActivationListForUserRequest.java

```diff
@@ -19,18 +19,16 @@
 package com.wultra.security.powerauth.client.model.request;
 
 import jakarta.validation.constraints.Min;
-import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
-import lombok.NoArgsConstructor;
 
 /**
  * Model class representing request for activation list for a given user.
  *
  * @author Petr Dvorak, petr@wultra.com
  */
 @Data
-@AllArgsConstructor
-@NoArgsConstructor
+@Builder
```

Comment:
One more thing, this annotation creates an all-arg package private constructor and you are not able to call `new GetActivationListForUserRequest()` anymore. Is it on purpose?